tasks/db_deploy/migrate_db.py

```diff
@@ -23,26 +24,86 @@ def perform_migration(current_schema_version: int, config: Dict[str, str]) -> No
     # Determine migrations to run based on current_schema_version and
     # update the versions table based on the latest_schema_version.
     if current_schema_version == 1:
-        # Run migrations from version 1 to the latest version
-        # in this case version 2 is the latest so we set the latest version
+        # Run migrations from version 1 to version 2
+        migrate_versions_1_to_2(config, False)
+        current_schema_version = 2
+        # Run migrations from version 2 to version 3
+        migrate_versions_2_to_3(config, True)
+
+    elif current_schema_version == 2:
+        # Run migrations from version 2 to the latest version
+        # in this case version 3 is the latest so we set the latest version
         # flag to True
-        migrate_versions_1_to_2(config, True)
+        migrate_versions_2_to_3(config, True)
 
+@retry_operational_error(MAX_RETRIES)
```

Comment:
Remove this, we only need it on a few key functions.

Reply:
removed